src/components/Table/EditableDraggableTable.js

```diff
@@ -0,0 +1,341 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DragDropContext } from 'react-dnd';
```

Comment:
this is a nice enhancement. It may make sense to contribute this to PF React in the future. thanks for the work on this!